### ISSUE TITLE
Korath v Quarg Engines Description: The Quarg have a weird definition of 'efficency'

### ISSUE THREAD

**Issue by marphod** [endless-sky/endless-sky#2735]
The description of the Korath Thrusters implies that they are inefficient and therefore distasteful to the Quarg.

Thing is, they are more efficient than the Quarg Thrusters. Both the Medium Graviton Thrusters and the Planetary Class Thrusters generate 288,000 units of thrust (Newtons? foot-pounds?  whatever).  The Quarg requires 70 tons of space, and consumes 1200 units of energy to do so and generates 600 units of waste heat. This is 240 units of Thrust per unit of energy, and 4,114 units of force per ton of space.  The Korath requires 69 tons of space and consumes 534 units of energy, producing 1362 units of waste heat, which is 539 units of Thrust per unit energy, and 4,174 units of thrust per ton.  They are dirtier, but way more efficient.  

(For steering, the Quarg are strictly worse than the tier 1/1.5 culture versions -- The Bondir Atomic Steering are just  slightly more efficient than the human equivalents, generating 106k units of turning force in 49 tons of space and for 192 units of energy, with 492 units of waste energy.  The Graviton Steering uses 1 ton more space, 300 units of energy, and generates 96k units of turning force wth 300 units of waste energy.  Which for a tier 4+ culture is just sad.)

tl;dr: Quarg engines are cleaner, but way less efficient than nearly every other culture's engines. (ignoring ion and plasma)

**Comment by Bladewood** [comment]
Quarg gear will be rebalanced later.

**Comment by marphod** [comment]
I went a bit afield -- my main point is that the Korath drives aren't that inefficient, compared to the other alternatives at approximately the same mass -- if you assume a logarithmic relationship between energy in and thrust out, the korath are near the expected curve. Why are the Quarg so notably unimpressed with the korath

**Comment by Bladewood** [comment]
They are not necessarily comparing the Korath to themselves, but most likely to other species the Quarg have met.

**Comment by Bladewood** [comment]
The Korath are close to the curve, but not close enough.

**Comment by Amazinite** [comment]
>Which for a tier 4+ culture is just sad.

Tier 3*

You're comparing a medium thruster to a large thruster. The Planetary Thruster is the Korath's large thruster. Their medium thruster is the Lunar thruster.

Here's my analysis:
Medium Graviton Thruster: 
* 4,114.29 thrust/ton
  * Best medium thruster for thrust efficiency
  * When compared to large thrusters, 2nd best for thrust efficiency (best being the Planetary Thruster)
  * 1.44% worse than the Planetary Thruster and 10.9% better than the Lunar Thruster
* 240.00 thrust/energy
  * Worst medium thruster for energy efficiency
  * When compared to large thrusters, worst for energy efficiency
  * 124.7% worse than the Planetary Thruster and 110.2% worse than the Lunar Thruster
* 480.00 thrust/heat
  * 2nd best medium thruster for heat efficiency (best being the Pug Cormet Thruster)
  * When compred to large thrusters, 2nd best for heat efficiency (best being the Pug Lohmar Thruster)
  * 127% better than the Planetary Thruster and 131.1% better than the Lunar Thruster

So for the Medium Graviton Thruster in terms of efficiency, the only place where it isn't highly efficient is in the case of energy efficiency. Although, the Quarg might not exactly consider this as inefficient since their energy generation is so efficient (not only in energy/ton but also energy/heat, unlike the Korath energy generation) that it negates the fact that the Medium Graviton Thruster eats up energy like crazy. The Korath might do moderately well in terms of how energy efficient they are, but they are the worst when it comes to heat efficiency in not only their engines but their energy generation too. The Quarg don't even worry about heat so much that they don't have cooling outfits. It's also important to note that when comparing the Graviton Thruster to Korath engines, it is more efficient comparatively in terms of thrust/heat than it is worse in thrust/energy (e.g. 127% better in thrust/heat vs 124.7% worse in thrust/energy when compared to the Planetary Thruster), so one can look at it in such a way that Graviton Thrusters are more efficient than Korath Thrusters, at least overall.

Medium Graviton Steering:
 * 1,920.00 turn/ton
   * 4th best medium steering for turn efficiency
   * When compared to large steering, 5th best for turn efficiency
   *  24.4% worse than Planetary Steering and 10% worse than Lunar Steering
 * 160.00 turn/energy
   * Worst medium steering for energy efficiency
   * When compared to large steering, worst for energy efficiency
   * 181.2% worse than Planetary Steering and 164% worse than Lunar Steering
 * 320.00 turn/heat
   * 2nd best medium steering for heat efficiency (best being the Pug Cormet Steering)
   * When compared to large steering, 2nd best for heat efficiency (best being the Pug Lohmar Steering)
   * 119.6% better than Planetary Steering and 124.2% better than Lunar Steering

While the Medium Graviton Steering is not as impressive in terms of turn efficiency as its thruster outfit is in terms of thrust efficiency, it still follows the same trend for energy and heat efficiency; worst for energy efficiency and 2nd best for heat efficiency, beat out by only the Pug. Although when compared to Korath steering, it isn't more heat efficient than it is worse energy efficient like Graviton Thrusters are. So if we're talking about Graviton Steering, then it isn't more efficient overall than Korath steering. But interestingly, it's only the Korath thruster descriptions which make mention of the Quarg considering them inefficient, so really, it all checks out.

**Comment by marphod** [comment]
You can't compare culture to culture outside of mass-of-engine comparisons.  The korath, human plasma and human atomics have five classes of engines, human ions and hai have 6 (although one of each is a hybrid thruster/steering engine), the Wanderers and pug 4, coalition 2, and quarg 1.

the definition of the Efficiency of an engine is defined as how much mechanical force you get per unit of energy used, compared to a theoretically loss-less energy conversion.  This isn't a matter of debate.  

However, after doing some mathematical analysis, I withdraw this.  While the korath engines are by far the most powerful in their weight class (8-12%better than the Hai, there closest competitor), ignoring the need for a massive quarg re-balance, they are less energy efficient than i thought, being 10% less efficient than human atomics, the next worse engines in the game.  (Korath atomcs and hai are all around 450-650 units of thrust per energy depending on engine size, the others are mostly 600-800, and the coalition is amazingly efficient at 860-924, although their engines are impractically large for the amount of thrust generated)

**Comment by Amazinite** [comment]
>the definition of the Efficiency of an engine is defined as how much mechanical force you get per unit of energy used, compared to a theoretically loss-less energy conversion. This isn't a matter of debate.

It is a matter of debate; you're completely leaving thrusting heat out of the equation. Heat and energy are both detriments to the efficiency of an engine, so you can't just look at thrust/energy.

**Comment by jafdy** [comment]
He's talking about the physics definition, I expect that the definition would be different in spacecraft engineering due to heat dissipation being a major problem.
